### PR TITLE
admission webhook: add README.md

### PR DESCRIPTION
**Pull request by derekbit**
Longhorn 3752

Signed-off-by: Derek Su <derek.su@suse.com>

### CODE REVIEW

File: webhook/admission/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Admission Webhook
+
+The admission webhook validates or changes the values in a request.
+An resource request from longhorn-manager might be handled by an
+admission webhook on another node. Therefore, any manipulation of
+the resources of a node, such as creating and deleting folders,
+should be avoided. The manipulations should be handled by the 
+longhorn-manager daemon or controller.
```
Comment (innobead) [inline webhook/admission/README.md:8]: How about

`
The admission webhook only validates or mutates the spec of hooked resources before resource reconsilation. Beyound that, any resource reconsilation to ensure resources behave as expected should still be handled in the backend (ex: controllers or even host operations like creating/deleting folders at host) as usual. 
`

**Review by innobead — CHANGES_REQUESTED** [review]

File: webhook/admission/README.md (hunk 1 above)
Comment (derekbit) [reply webhook/admission/README.md:8]: Looks good, thank you.

Comment (innobead) [reply webhook/admission/README.md:8]: LGTM

File: webhook/admission/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Admission Webhook
+
+The admission webhook only validates or mutates the spec of hooked resources
+before being persistent to the database like etcd. Any resource reconciliation
+to ensure resources behave as expected should be handled in the backend as usual.
+For exampleex, controllers or even host operations like creating and deleting
```
Comment (innobead) [inline webhook/admission/README.md:6]:
```suggestion
For example, controllers or even host operations like creating and deleting
```

**Review by innobead — APPROVED** [review]

**Pull request merged**